aimmo-game/simulation/turn_manager.py

```diff
@@ -20,19 +21,20 @@ def __init__(self):
         self._world_state = None
```

Comment:
an artifact from renaming `WorldStateManager` to the (imo) less confusing `GameStateManager`